infrastructure/dpladm/vpa-templates/cli-vpa.template.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     containerPolicies:
     - containerName: cli
       minAllowed:
-        cpu: 4m
-        memory: 2Mi
+        cpu: 200m
+        memory: 200Mi
       maxAllowed:
-        cpu: 150m
-        memory: 250Mi
+        cpu: 400m
+        memory: 1.1Gi
```

Comment:
Is there a reason why this is raised to 1.1Gi when we requested 1Gi?

It is not the end of the world. I am just wondering :smile:

Reply:
> recommand killing the pod

Recommend? Then who takes de decision?